src/Config/OssIndexServerConfig.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Config } from "../Types/Config";
+import { writeFileSync, readFileSync } from "fs";
+
+export class OssIndexServerConfig extends Config {
+  constructor(readonly username: string, readonly token: string) {
+    super(username, token);
+  }
+
+  public saveConfigToFile(
+    saveLocation: string = this.getSaveLocation()
+  ): boolean {
+    let ableToWrite = false;    const stringToSave = `Username: ${this.username}
```

Comment:
Newline?

Reply:
Also I'd make a quick lil `getStringToSave(): string` helper method rather than do this.

Reply:
@DarthHater oh! not done, just making a draft PR for now!

Reply:
in: https://github.com/sonatype-nexus-community/auditjs/pull/98/commits/aace2d4a68211f9d1c8d1acb689776cfa49477b5